sdk/src/main/java/com/horizen/utils/Ed25519.java

```diff
@@ -0,0 +1,43 @@
+package com.horizen.utils;
+
+import org.bouncycastle.crypto.params.Ed25519PrivateKeyParameters;
+import org.bouncycastle.crypto.params.Ed25519PublicKeyParameters;
+import scorex.crypto.hash.Sha256;
+
+public class Ed25519 {
+
+    public static int keyLength() {
+        return org.bouncycastle.math.ec.rfc8032.Ed25519.SECRET_KEY_SIZE;
```

Comment:
why not to import `org.bouncycastle.math.ec.rfc8032` or `org.bouncycastle.math.ec.rfc8032.Ed25519 => <alias>`